server/scripts/examples-gg-latest.js

```diff
@@ -67,7 +67,7 @@ canvas {
 
 const headers = { 'User-Agent': 'p5js-web-editor/0.0.1' };
 
-const mongoConnectionString = process.env.MONGO_URL;
+const mongoConnectionString = process.env.MONGO_URL || process.env.MONGO_URI;
```

Comment:
is there any reason for having these two environment variables? this seems confusing.

Reply:
So, MongoLab has chosen to set the Mongo DB URLs into environment in a variable called MONGO_URI, instead of MONGO_URL. Putting both in was a quick way to get it working...

But I'll see if maybe I can script it so that the deployment can copy the value of MONGO_URI into MONGO_URL so we can go back to just the one variable.

Reply:
got it. yeah, if possible it would be nice not to have two different environment variables that have almost the same name that reference the same thing.